### PR TITLE
Sanity Check 1

### PR DESCRIPTION
**Pull request by gb92**
Gavin Benedict
34437459

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "doorman"]
```
Comment (pragdave) [inline .gitmodules:1]: Just wondering: why is this a submodule and not a hex dependency?

**Comment by pragdave** [comment]
Looking forward to the final project.

File: .gitmodules (hunk 1 above)
Comment (gb92) [reply .gitmodules:1]: I was following their documentation in their git readme, which matched the status of their repo but not their published package. This unblocked me, but I might get rid of the code that requires the newer version, or switch once they publish

Comment (gb92) [reply .gitmodules:1]: Though now reading through the hex docs it looks like I can just reference a GitHub there. Will change that

**Pull request closed without merging**